FAQ.md

```diff
@@ -0,0 +1,49 @@
+# FAQ
```

Comment:
I suggest to add table of content and check the subtitle to question form
Why my quantized model has the same size as the original model ?
How to load quantized model?
Why I'm getting "torch.fx" parsing model error?